addons/web/static/src/views/kanban/kanban_record.js

```diff
@@ -69,7 +69,7 @@ function getColorName(value) {
  *  exist
  * @returns {string}
  */
-function getImageSrcFromRecordInfo(record, model, field, idOrIds, placeholder) {
+export function getImageSrcFromRecordInfo(record, model, field, idOrIds, placeholder) {
```

Comment:
Is it better to have all the exported function here or in the `views/utils.js` ?